app-admin/updateservicectl/updateservicectl-9999.ebuild

```
@@ -2,22 +2,22 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-CROS_WORKON_PROJECT="flatcar-linux/updateservicectl"
+CROS_WORKON_PROJECT="kinvolk/updateservicectl"
 CROS_WORKON_LOCALNAME="updateservicectl"
 CROS_WORKON_REPO="git://github.com"
-COREOS_GO_PACKAGE="github.com/coreos/updateservicectl"
-COREOS_GO_GO111MODULE="off"
+COREOS_GO_PACKAGE="github.com/kinvolk/updateservicectl"
+COREOS_GO_GO111MODULE="on"
 inherit cros-workon coreos-go
 
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm64"
 else
-	CROS_WORKON_COMMIT="b0e1d578b42082b5bb5ccb1b6e6d526843df3f46" # v2.1.0
+	CROS_WORKON_COMMIT="6a4ff4ca879082c07353dd379439c437cbe27e18" # v2.1.0
```

```suggestion
	CROS_WORKON_COMMIT="6a4ff4ca879082c07353dd379439c437cbe27e18" # main
```

sneaky change - commit ID is no more associated to a release tag :/

---

done